src/components/Components/LinkedSecretsListView/LinkedSecretsListView.tsx

```diff
@@ -147,20 +171,24 @@ export const LinkedSecretsListView: React.FC = () => {
             <Text component={TextVariants.p}>Linked secrets</Text>
           </TextContent>
 
-          <Table
-            virtualize={false}
-            data={sortedLinkedSecrets}
-            unfilteredData={sortedLinkedSecrets}
-            NoDataEmptyMsg={NoDataEmptyMessage}
-            aria-label="Linked Secrets List"
-            Header={LinkedSecretsListHeader}
-            Row={LinkedSecretsListRow}
-            loaded={linkedSecretsLoaded}
-            getRowProps={(obj: SecretKind) => ({
-              id: `${obj.metadata.name}-linked-secret-list-item`,
-              'aria-label': obj.metadata.name,
-            })}
-          />
+          <div data-test="linked-secrets-list">
```

Comment:
why we need the div?

Reply:
I'm using that for the tests:

```ts
const linkedSecretsList = screen.getByTestId('linked-secrets-list');
```

but if there's a better approach, I'm all ears to know about it 🙃

Reply:
would we get the table for testing by aria-label? i see it has been there?

Reply:
cool! testing by aria-label worked! 

thanks for the suggestion! :)